src/routes/page.scss

```diff
@@ -0,0 +1,18 @@
+.main-image img {
+  max-height: 450px;
+}
+
+.construction-sign {
+  width: 100%;
+  max-width: 175px;
+}
+
+.construction-sign-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.homepage-greeting {
+  word-wrap: anywhere;
```

Comment:
this was to deal with the url. I imagine that we're going to avoid raw urls for the most part, but perhaps this is something to add at least to the link component to handle this on smaller widths?

Reply:
Made a TODO and can potentially create a small ticket to capture this somewhere